### ISSUE TITLE
[bug] position of model in full scree mode

### ISSUE THREAD

**Issue by koebel** [saw-jan/web-app-3dmodel-viewer#8]
When switching between fullscreen mode and standard mode, I would expect that the model stays in the center of the viewport. However the model stays in original position which is off center in full scree mode. 

Tested on MacOS 10.14.6 with Chrome Version 116

**Comment by saw-jan** [comment]
>Not sure if this relates to the same problem

Yeah, currently the viewport won't change with window resize so the initialize size is maintained.
Needs a fix
